Turbolinks/VisitableView.swift

```diff
@@ -132,12 +132,21 @@ public class VisitableView: UIView {
             screenshot.translatesAutoresizingMaskIntoConstraints = false
             screenshotContainerView.addSubview(screenshot)
             
-            NSLayoutConstraint.activateConstraints([
-                screenshot.centerXAnchor.constraintEqualToAnchor(screenshotContainerView.centerXAnchor),
-                screenshot.topAnchor.constraintEqualToAnchor(screenshotContainerView.topAnchor),
-                screenshot.widthAnchor.constraintEqualToConstant(screenshot.bounds.size.width),
-                screenshot.heightAnchor.constraintEqualToConstant(screenshot.bounds.size.height)
-            ])
+            if #available(iOS 9.0, *) {
```

Comment:
You can ditch this whole `if/else` statement and just use the iOS 8.0 method since the behavior is the same